extension.ts

```diff
@@ -114,6 +115,15 @@ export async function activate(context: vscode.ExtensionContext) {
 
   vscode.window.onDidChangeActiveTextEditor(handleActiveEditorChange, this);
 
+  vscode.workspace.onDidChangeTextDocument((event) => {
+    /* TODO: Remove setTimeout (https://github.com/Microsoft/vscode/issues/11339) */
```

Comment:
Can you write a little documentation on why this is needed (isDirty not immediately set etc)